section_1_review.livemd

```diff
@@ -183,18 +210,21 @@ Update (increment) the `:count` key in this map to return `%{count: 2}`.
 
 ```elixir
 %{count: 1}
+|> Map.update!(:count, &(&1 + 1))
```

Comment:
Pipe operator is fun, but generally, tools like credo will warn you when using a single pipe. The code is arguably more clear when written as:

```elixir
Map.update!(%{count: 1}, :count, &(&1 + 1))
```